analyses/molecular-subtyping-integrate/README.md

```diff
@@ -11,6 +11,11 @@ bash run-subtyping-integrate.sh
 
 ### Module contents
 
+`00-prepare-extraction-table.Rmd`creates a table `pbta_extraction_status.tsv` of RNA samples with `extraction_type` as one of the following:
```

Comment:
If you're going to use the histologies file, I'd recommend adding this to `run-subtyping-integrate.sh` as well

```
Rscript -e "rmarkdown::render('00-prepare-extraction-table.Rmd', clean = TRUE)"
```

Do I expect we will need to run this again? And do I expect it would change even if we did? I hope not on both points, but if we need to rerun this for whatever reason, I'd expect the bash script to run both steps _unless_ you explicitly state that it won't in the docs. So that's the other option – write a note under the usage section that this not included in the bash script.

Reply:
yes, will add.